src/axom/core/utilities/AnnotationMacros.hpp

```diff
@@ -85,15 +86,16 @@
  * 
  */
 #if defined(AXOM_USE_ANNOTATIONS) && defined(AXOM_USE_CALIPER)
-#error "Support for Caliper has not yet been implemented in Axom!"
+  #error "Support for Caliper has not yet been implemented in Axom!"
 #elif defined(AXOM_USE_ANNOTATIONS)
-#define AXOM_PERF_MARK_SECTION(__name__, ... ) \
-        AXOM_NVTX_SECTION( __name__ , __VA_ARGS__ )
+  #define AXOM_PERF_MARK_SECTION(__name__, ...) \
+    AXOM_NVTX_SECTION(__name__, __VA_ARGS__)
 #else
-#define AXOM_PERF_MARK_SECTION(__name__,...) \
-  do {                                      \
-    __VA_ARGS__                             \
-  } while( false )
+  #define AXOM_PERF_MARK_SECTION(__name__, ...) \
+    do                                          \
+    {                                           \
+      __VA_ARGS__                               \
+    } while(false)
```

Comment:
It does a really nice job with nested compiler pragmas and with aligning internal line breaks